alembic/autogenerate/rewriter.py

```diff
@@ -23,6 +23,10 @@
     from ..runtime.environment import _GetRevArg
     from ..runtime.migration import MigrationContext
 
+ProcessRevisionDirectiveFn = Callable[
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4990